.github/workflows/comprehensive-testing.yml

```
@@ -0,0 +1,603 @@
+    timeout-minutes: 5
+
+    steps:
+      - name: Notify Slack on Failure
+        if: env.SLACK_WEBHOOK_URL != ''
+        uses: 8398a7/action-slack@v3
+        with:
+          status: failure
+          channel: "#ci-cd"
+          username: "GitHub Actions"
+          icon_emoji: ":x:"
+          title: "CI Pipeline Failed"
```

actionlint (1.7.7)

input "title" is not defined in action "8398a7/action-slack@v3". available inputs are "author_name", "channel", "custom_payload", "fields", "github_base_url", "github_token", "icon_emoji", "icon_url", "if_mention", "job_name", "mention", "status", "text", "username"

(action)